samples/cellular/nrf_cloud_rest_device_message/boards/nrf9160dk_nrf9160_ns.overlay

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
We need to add support for the 9161 to all our samples. I've created a ticket for it: https://nordicsemi.atlassian.net/browse/IRIS-6848.

When that happens, there will need to be an overlay for that as well here, I assume.